mkdocs.dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
Nit: I would prefer to move `mkdocs.Dockerfile`, `mkdocs.requirements.txt`, and `docs.mk` to a sub-directory to clean-up the main directory. Somewhere like  `hacks/api-docs` or `docs`